### PR TITLE
[CodeClean] combine if statements.

### PR DESCRIPTION
**Pull request by mojunsang26**
combine if statements in tensor_transform.c.
I confirmed it worked well in example_image_classification_tflite.
related isuue: #1786 

Signed-off-by: Junsang Mo <junsang.mo@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_transform/tensor_transform.c [hunk 1]
```diff
@@ -1478,55 +1478,37 @@ gst_tensor_transform_convert_dimension (GstTensorTransform * filter,
...
-          } else if (i > a && i > b) {
-            out_info->dimension[i] = in_info->dimension[i];
-          } else if (a > b) {
-            if (i == b) {
-              out_info->dimension[i] = in_info->dimension[a];
-            } else {
+          } else if (i == b) {
+            out_info->dimension[i] = in_info->dimension[a];
+          } else {
+	    if (a > b) {
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_transform/tensor_transform.c]: indent? BTW, about about using `else if`

**Comment by myungjoo** [comment]
Maybe the script "gst-indent" can save your effort.

File: gst/nnstreamer/tensor_transform/tensor_transform.c (hunk 1 above)
Comment (mojunsang26) [reply gst/nnstreamer/tensor_transform/tensor_transform.c]: Thank you for your support! Let me reflect your review.

**Comment by mojunsang26** [comment]
> Maybe the script "gst-indent" can save your effort.
Thank you for your tip! It is very convenient!

**Pull request merged**